requirements.txt

```diff
@@ -10,7 +10,8 @@ simplejson==3.19.3
 psycopg2-binary
 sentry-sdk[django]==2.19.2
 python-dateutil==2.9.0.post0
-discord-webhook[async]
+# discord-webhook[async]
+git+https://github.com/cemathey/python-discord-webhook@132fc7b7a58ba25e7fffd71cb363e4748c82941c
```

Comment:
Can you make a PR for the necessary changes upstream, so we can eventually go back to the official one? :)